recipe/meta.yaml

```diff
@@ -8,59 +8,50 @@ package:
 source:
   url: https://github.com/libsndfile/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.xz  
   sha256: 3799ca9924d3125038880367bf1468e53a1b7e3686a934f098b7e1d286cdb80e  
-  patches:
-    # the libs private sections of pkgconfig is only useful when statically
-    # linking which we do not use.
-    # On windows, this results in an empty "-l" which breakds downstream packages
-    # since the linker tries to link to ".libs"
-    # https://github.com/conda-forge/gst-plugins-bad-feedstock/pull/20#issuecomment-1336553304
-    # it may be related to
-    # https://github.com/libsndfile/libsndfile/issues/827
-    # But I don't have time to investigate too much
-    - disable_libs_private_pkgconfig.patch
-
 
 build:
-  number: 1
+  number: 0
   run_exports:
         # https://abi-laboratory.pro/?view=timeline&l=libsndfile
     - {{ pin_subpackage(name, max_pin="x.x") }}
+  skip: true  # [s390x or win]
 
 requirements:
   build:
     - gnuconfig  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make   # [not win]
-    - cmake  # [win]
-    - ninja  # [win]
     - python
     - pkg-config  # [unix]
   host:
-    - libflac
-    - libvorbis
-    - libogg
-    - libopus
-    - lame
-    - mpg123
+    - libflac 1.4.3
+    - libvorbis 1.3.7
+    - libogg 1.3.5
+    - libopus 1.3
+    - lame 3.100
+    - mpg123 1.30.0
+  run:
+    - {{ pin_compatible('mpg123') }}
```

Comment:
We probably want to be more strict than that. 1.25.8 has `lib/libmpg123.so.0.44.7` while 1.30.0 has `lib/libmpg123.so.0.47.0`. So I don't know how they version their DSO files, but they don't necessarily seem to follow a logical pattern.

Reply:
More strict as in, we probably want to put an upper bound on the minor version just to be safe.

Reply:
Though, https://abi-laboratory.pro/?view=timeline&l=mpg123 is quit clean... Maybe an upper bound on the major is enough...